src/pybind/mgr/dashboard/controllers/user.py

```diff
@@ -12,6 +13,37 @@
 from ..services.auth import JwtManager
 
 
+# minimum password complexity rules
+def check_password_complexity(password, username):
+    """
+    Suggested rules for password complexity
+    - at least 6 chars in length
+    - must not be the same as the user account name
+    - consist of characters from the following groups
+    - alphabetic a-z, A-Z
+    - numbers 0-9
+    - special chars: !_@
+    - must use at least 1 special char
+    """
+    if password == username:
```

Comment:
Does this work in a vstart environment that uses "admin/admin" by default?

Reply:
yes, because this method is called only during creating/editing user in Ceph Dashboard

Reply:
Maybe we should stop using `admin/admin` by default and using an auto-generated password? That's what the `restful` module does in vstart, for example. In any case, if we are concerned about security, having a fixed password for `admin` account should be the exception (if any) rather than the rule.